CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
+## 5.1.0 (2020-09-17)
+
+* Remove uploadDebugBuildMappings flag in favour of variantFilter
+  [#301](https://github.com/bugsnag/bugsnag-android-gradle-plugin/pull/301)
+
+* Fix task ordering for upload task on AGP 3.5
+  [#300](https://github.com/bugsnag/bugsnag-android-gradle-plugin/pull/300)
+
 ## 5.0.2 (2020-09-10)
 
 * Reword log messages that implied duplicate uploads
   [#296](https://github.com/bugsnag/bugsnag-android-gradle-plugin/pull/296)
 
+* Fix bugsnag.sharedObjectPaths not uploading SO files
+  [#295](https://github.com/bugsnag/bugsnag-android-gradle-plugin/pull/295)
```

Comment:
This be in 5.1.0, shouldn't it?

Reply:
Yes - fixed this to go in the right section.